src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidatorTest.java

```diff
@@ -865,6 +866,66 @@ void testErrorReportedForMissingUpdateEntityNameField() throws ServiceException
         assertError(qualifiedFieldName, validationMessage, errors);
     }
 
+    @Test
+    void testNotErrorReportedForHasSecureRegisterFieldSetToFalseForFullValidation() throws ServiceException {
```

Comment:
I've now included 'Registration' in the names of the test methods I did and added a couple of tests for Update too. Please take a look.